buildroot-external/rootfs-overlay/usr/lib/systemd/system/mnt-data.mount

```diff
@@ -5,6 +5,8 @@ DefaultDependencies=no
 After=hassos-expand.service
 Before=umount.target local-fs.target
 Conflicts=umount.target
+After=systemd-fsck@dev-disk-by\x2dlabel-hassos\x2ddata.service
```

Comment:
Is it allowed to have multiple `After=` lines? There is already one above.

Reply:
Hm, description is not very clear, but I think so? https://www.freedesktop.org/software/systemd/man/systemd.syntax.html#

Reply:
Yea it seems it is fine to have multiple.

Reply:
both work